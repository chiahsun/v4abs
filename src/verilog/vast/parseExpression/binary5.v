b2[3:2]|a2[1:0];
