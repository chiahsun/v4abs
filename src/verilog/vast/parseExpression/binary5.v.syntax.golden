(expression (binary_expression (binary_expression_bw_or (expression (primary (select_identifier (identifier (single_identifier (simple_identifier b2))) (one_or_more_select (select (range_select (expression (primary (number (unsigned_number 3)))) (expression (primary (number (unsigned_number 2)))))))))) (expression (primary (select_identifier (identifier (single_identifier (simple_identifier a2))) (one_or_more_select (select (range_select (expression (primary (number (unsigned_number 1)))) (expression (primary (number (unsigned_number 0)))))))))))))
